packages/core/src/base-chart.ts

```diff
@@ -486,12 +486,14 @@ export class BaseChart {
 	}
 
 	addLegend() {
-		if (this.container.select(".legend-tooltip").nodes().length > 0) {
+		if (this.container.select(this.options.rtl ?  ".legend-tooltip-rtl" : ".legend-tooltip").nodes().length > 0) {
```

Comment:
Again we should have a more efficient way to do this

Reply:
rtl could be another class that's added, and `legend-tooltip` we can always have in the classname so that we won't care about whether rtl is on or not to find the legend tooltip element